jujugui/static/gui/src/app/models/models.js

```diff
@@ -2713,16 +2713,16 @@ YUI.add('juju-models', function(Y) {
     _collapseMachineConstraints: function(constraints) {
       var constraint = '';
       var constraintMap = {
-        arch: 'arch',
+        availabilityZone: 'availability-zone',
         cpuCores: 'cpu-cores',
         cpuPower: 'cpu-power',
-        mem: 'mem',
         disk: 'root-disk'
```

Comment:
Oh, ok, I thought it was a list of supported constraints.  That might be a nice place for a comment, since it was previously being used wrong.